tests/scr/sindTestContainerInit

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash -e
```

Comment:
Not sure i understand why is this script needed. The SinD folder already have an initialization script for this purpose. Let's sync-up later.

Reply:
This is the script to initialize the test container inside of which the sysbox-in-docker container runs. The initialization script you are describing is for the sysbox-in-docker container itself.

Reply:
Got it.